src/protocols/ethereum/config.ts

```diff
@@ -26,29 +27,21 @@ export const ETH_NETWORK_DEFAULT_SETTINGS: IDefaultNetworkTypeData<IEthNetworkSe
   },
 };
 
-export const ETH_NETWORK_ADDITIONAL_SETTINGS: IDefaultNetworkTypeData<any> = { // TODO - type
+export const ETH_NETWORK_ADDITIONAL_SETTINGS: IDefaultNetworkTypeData<
+  IEthNetworkPredefinedSettings
+> = {
   [NETWORK_TYPE_MAINNET]: {
     explorerUrl: 'https://etherscan.io',
     middlewareUrl: 'https://api.etherscan.io/api', // TODO replace temp values
+    tokenMiddlewareUrl: 'https://api.ethplorer.io/',
   },
   [NETWORK_TYPE_TESTNET]: {
     explorerUrl: 'https://sepolia.etherscan.io',
     middlewareUrl: 'https://api-sepolia.etherscan.io/api', // TODO replace temp values
+    tokenMiddlewareUrl: 'https://sepolia-api.ethplorer.io/',
```

Comment:
I think we should be consistent with the way we are adding such urls, both urls above have no `/` symbol in the end of it,

Reply:
You're right. Updated.